pages/api/content/glamorous.js

```diff
@@ -1,6 +1,131 @@
 module.exports = {
-  title: '`glamorous`',
+  title: 'glamorous API',
   subtitle: '',
-  description: ``.replace(/~/g, '`'),
+  description: `
+    The ~glamorous~ function allows you to create your own ~glamorousComponentFactory~ (see 
+    [above](https://github.com/paypal/glamorous/blob/master/README.md#glamorouscomponentfactory)
```

Comment:
In the glamorous docs the `glamorousComponentFactory` part is indeed above the `api` part, but on this website it is underneath of it. I don't know which one is correct, but I think a change is required here.